cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -202,6 +202,9 @@ EOF
     echo "Service hive-metastore is not loaded"
   fi
 
+  # Check that metastore schema is compatible.
+  /usr/lib/hive/bin/schematool -dbType mysql -info
```

Comment:
Can you add something like `|| err 'Run /usr/lib/hive/bin/schematool -dbType mysql -upgradeSchemaFrom 2.1.0 to upgrade the schema. Note that this may break Hive metastores that depend on the old schema'`

Reply:
Done